tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/message/RawMessage.kt

```diff
@@ -194,6 +194,7 @@ internal data class RawMessage(
             supergroup_chat_created -> SupergroupChatCreated(
                 migrate_from_chat_id
             )
+            migrate_from_chat_id is ChatId -> MigratedToSupergroup(migrate_from_chat_id)
```

Comment:
I think, we should change type of `migrate_from_chat_id` instead of class casting

Reply:
Done